.pytest_cache/v/cache/lastfailed

```diff
@@ -0,0 +1,173 @@
+{
```

Comment:
This file should not have been commited.

Reply:
I have removed it from commit

Reply:
Done